src/Image.php

```diff
@@ -295,6 +297,10 @@ public function listOperations()
     /**
      * Create a stack.
      *
+     * @deprecated 2.0.0
```

Comment:
I guess this line can be combined with the previous one

Reply:
true, fixed